src/components/cap-reporter.js

```diff
@@ -79,7 +83,7 @@ export default class CapReporter extends LitElement {
 				display: block;
 			}
 
-			.list__title {
+			.reporter__volumeTitle {
```

Comment:
Renamed this for consistency with BEM. An ement with a class of `list__title` needs to have a parent with the class of `list`. In this case, there is a parent with a class of `reporter` so I swapped it in.